public/theme/skeleton/index.twig

```diff
@@ -19,4 +19,13 @@
 
     {% endif %}
 
+    <hr>
```

Comment:
This is a revert of https://github.com/bolt/four/commit/bf0f9eab211181712d405e9fa279e2f942c1b21a#diff-f6a467f42f0eec03b12d623560b5d5aaL26 , and is not needed anymore

Reply:
removed